### PR TITLE
React signin tests

### PR DESCRIPTION
**Pull request by BenBagBag**
## Purpose

* Fixes React Signin tutorial tests so they're no longer broken.

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x  ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[  ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
cd 1-Authentication/2-sign-in-angular/SPA/
npm test
```

## What to Check
Verify that the following are valid
* All checks pass

### CODE REVIEW

**Review by iambmelt — APPROVED** [review]
LGTM but let's also get review from Hector before we merge.

**Review by hectormmg — APPROVED** [review]
LGTM

**Pull request merged**
